### PR TITLE
fix(server/functions): ban checking when no license2

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

Fixes an issue introduced in #519 where function catches when checking for a player without license2

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by solareon — APPROVED** [review]
Dealer's choice

File: server/functions.lua [hunk 1]
```diff
@@ -332,29 +332,27 @@ exports('ToggleOptin', ToggleOptin)
...
     if os.time() < result.expire then
         local timeTable = os.date('*t', tonumber(result.expire))
+
         return true, ('You have been banned from the server:\n%s\nYour ban expires in %s/%s/%s %s:%s\n'):format(result.reason, timeTable.day, timeTable.month, timeTable.year, timeTable.hour, timeTable.min)
     else
         CreateThread(function()
-            storage.deleteBan({
-                license = plicense
-            })
+            storage.deleteBan({ license = license2 })
```
Comment (mmoftah) [inline server/functions.lua:351]: These will cause the same issue as before since `deleteBan` calls `getBanId`

**Review by solareon — APPROVED** [review]

File: server/functions.lua (hunk 1 above)
Comment (solareon) [reply server/functions.lua]: Fixed this one with a guard clause

**Comment by mmoftah** [comment]
LGTM, tested locally and it lets me in 🎉

File: server/functions.lua [hunk 2]
```diff
@@ -332,29 +332,30 @@ exports('ToggleOptin', ToggleOptin)
...
     else
         CreateThread(function()
-            storage.deleteBan({
-                license = plicense
-            })
+            if license2 then
+                storage.deleteBan({ license = license2 })
+            end
+
+            storage.deleteBan({ license = license })
```
Comment (Manason) [inline server/functions.lua:355]: This code implies that license2 is optional, but license is guaranteed. Is that the case in FiveM?

Comment (solareon) [reply server/functions.lua:355]: sadly that's the case. i think it's probably a relic from folks with a REALLY old install of fivem that they just don't get a license2 created at some point

**Pull request merged**